### ISSUE TITLE
Community Highlighted Servers

### ISSUE THREAD

**Issue by mirasrael** [StrangeLoopGames/EcoIssues#21622]
We need way to promote servers which we can recommend to new players. Such servers should have long enough history, stable online and good hardware to be able to handle big groups of players without lags. We can define criteria like these and then show servers on separate recommended tab or highlight somehow.
This way we can guarantee new players will have good experience and server admins who puts lot of effort in their server will be highlighted.
Unfortunately many servers runs just for few days and then wipe or shutdown. Not best place for starting experience. We have official servers which servers same purpose, but there lot of community servers which may offer different experience to players (some modded, some have interesting non-standard server settings, some have different economics etc).

**Comment by IronyTroll** [comment]
This suggestion closely resembles another suggestion, I wholeheartedly agree! Merited Servers #1413

**Comment by MishkaFur** [comment]
yes please, first impressions are a huge part of the game.

I was lucky when I frst started playing that the server was stable, subtley modded and had good admin but whilst hopping between other servers since then before running my own I have ended up on some that  admin has spawned in skids for themselves, devtooling half the planet etc.

Could it be worth having like the official servers section of "Your worlds" but it just rotates through a list of X number of the merited/highlighted servers if not a a section on server browser that show Official and Merited servers side by side Perhaps server icon to distinguish them to ensure players do not mistake them for official in any form or manner.

I was speaking to Irony when he did his github suggestion and "Merited" came up as a more suitable name than others and it also fits in with the discord name of Merited Citizen for someone who has been very helpful to the community.

**Comment by MonzUn** [comment]
The first time I played online, the server owner killed off the population on day 5 or so because they spawned themselves stuff and just answered that "I pay for the server so I can do what I want" when confronted about it.

I only got back into the game because a few friends who hadn't played it before wanted to give it a go. Ended up on a way better server where I quickly became a mod and met some of the moderators that work with me on my server now. But it was still just by chance that I gave the game a second shot.

The first impressions are super important and from my own experience I realized how important it is for server growth that people who come in feel welcome and that they have a place. Both the server and the user benefits from this so it's a win-win :)

**Comment by Dave247** [comment]
SLG would need to make a decision on how much control and oversight they want to have on such a system, keeping in mind that choosing to promote one server over another can be seen as a endorsement of that server from SLG themselves.

One method that FPS games use is server licencing, which can vary again depending on how much control the developers want to retain over the system. For example [here is a link that explains how server licencing works for Squad.](https://squad.fandom.com/wiki/Server_licensing)
The process essentially boils down to a simple vetting process that is quite flexible, as it allows communities to rent a server from a dedicated host provider as well as run the dedicated server themselves on their own hardware if they so wish.

**Comment by Praefator** [comment]
I'm all for this! The main reason I even started the Maiestas**Haven** server, was because I was seeing left and right about how outright bad the server experience can be for new players. Hence the Haven part of the server name, to promote the sense of having a well-organized, stable server for people to see the benefits of collectivizing and working together towards the destruction of the meteor. 

Something which worked great for a very long time, from November 2018 (at the server's inception) up around the time 9.0 came around in September 2020, when we noticed just how much player dropoff there is by mid-game, let alone late game. Something which prevented us from legitimately build the (by then traditional) megaproject due to the sheer lack of manpower. Having a steady influx of players who will be shown the ropes (through a multitude of new player-oriented integration programs), while at the same time being helped all map long, will go a long way towards ensuring both server livelihood as well as its prospects towards becoming more of a longterm project instead of having to reset every 30-60 days to give people a fresh start and things to do. 

So long story short: as the players above me have mentioned: **first impressions last**, and what better way to garner interest in the game than by dropping new players in the able hands of veterans such as myself, and in turn have whole generations of players ready for whatever comes next in their respective journeys?

**Comment by ItsKodas** [comment]
### What I think should be added
### We should call them "Verified Servers"

**A)** There should be a form that server owners can fill out that would include things such as; how long the server has been running for (and the community), dedi specs and bandwidth, datacenter/provider (if applicable), server name / IP, discord server invite, contact details.

**B)** Servers must reach certain requirements, for example: just because a server is very popular or has the best system specs should not mean they receive a verified position, I think a representative from SLG should spend time in the communities that have applied to become a verified server that be their discord or in game server, communities should be an overall positive environment.

**C)** Verified server positions should not be an easy thing to receive and should be re-checked every couple months or so, because if a community essentially dies and stops working on their servers, they shouldn't stay verified.

**D)** When communities apply to become verified, SLG needs to determine whether or not that server meets requirements (population wise) relative to the region it is based in. As an example I am running a server in Australia that usually peaks around 12~ each day, where that may seem like absolutely nothing when comparing it to a European server or American server, its actually the biggest server in population when you compare it to everything else in the country.
Essentially servers should be graded by region and not globally otherwise certain countries with a dramatically smaller population will be essentially ignored.
